### PR TITLE
docs: Update Sass Variables docs to include a-la-carte req

### PR DESCRIPTION
**Pull request by cpitt**
## Description
Updated SassVariables.json doc file replacing info alert
"SCSS is used for variable files to support maps using indented syntax."
with warning alert about A-la-carte installation

## Motivation and Context

Addresses #8366

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [x] My code follows the code style of this project.
- [x] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)
- [x] I've added new examples to the kitchen (applies to new features and breaking changes in core library)

### CODE REVIEW

**Comment by cpitt** [comment]
Note: the current Alert 
> SCSS is used for variable files to support maps using indented syntax.

didn't seem particularly useful. Happy to add an additional warning instead of replacing it if anyone disagrees

File: packages/docs/src/lang/en/customization/SassVariables.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "header": "# SASS variables",
   "headerText": "Starting with version **2.0**, Vuetify is built using **SASS/SCSS**. Using the data option in your `vue.config.js`, we can also pass in custom variables. A list of available variables is located in each components' folder. In order to build the SASS file, you will need to configure your application to support [SASS](https://sass-lang.com/dart-sass) and [Fibers](https://github.com/laverdet/node-fibers).",
-  "sassAlert1": "SCSS is used for variable files to support maps using indented syntax.",
-  "sassHeader2": "## Setup vue.config.js",
+  "sassAlert1": "Note: SCSS/SASS vrriables are only supported using [A-la-carte installation](https://vuetifyjs.com/en/customization/a-la-carte).",
```
Comment (johnleider) [inline packages/docs/src/lang/en/customization/SassVariables.json]:
```suggestion
  "sassAlert1": "Note: SCSS/SASS vrriables are only supported using [A-la-carte installation](/customization/a-la-carte).",
```

Comment (nekosaur) [reply packages/docs/src/lang/en/customization/SassVariables.json]: `variables` not `vrriables`

Comment (cpitt) [reply packages/docs/src/lang/en/customization/SassVariables.json]: Caught that after I accepted the suggestion, fix is already in the diff

**Pull request merged**